src/VideoManager/VideoManager.cc

```diff
@@ -305,7 +305,30 @@ VideoManager::sendTarget(double x, double y, double width, double height, double
     qCDebug(VideoManagerLog) << "maxX: " << maxX;
     qCDebug(VideoManagerLog) << "maxY: " << maxY;
 
-    qgcApp()->showAppMessage(QString("Target ROI: x=%1 y=%2 w=%3 h=%4").arg(x).arg(y).arg(width).arg(height));
+    qgcApp()->showAppMessage(tr("Target ROI: x=%1 y=%2 w=%3 h=%4").arg(x).arg(y).arg(width).arg(height));
+    _sendTargetViaMavlink(x, y, width, height, maxX, maxY);
+}
+
+void
+VideoManager::_sendTargetViaMavlink(double x, double y, double width, double height, double maxX, double maxY)
+{
+    if (!_activeVehicle) {
+        qCWarning(VideoManagerLog) << "Active Vehicle is not defined";
+        qgcApp()->showAppMessage(tr("Active Vehicle is not defined"));
+        return;
+    }
+
+    _activeVehicle->sendCommand(
+        _activeVehicle->compId(),         // Target component
```

Comment:
I guess should be ID of companion computer.

Reply:
agree - done